### PR TITLE
Improve table & columns description formatting (#98)

### PR DESCRIPTION
**Pull request by Mikhail-Ivanov**
### Summary of Changes

Introduced support of Markdown descriptions for tables & columns.
This module changes are intended to pass description text as a request body (instead of using URI parameter).

[Related changes at amundsenfrontendlibrary.](https://github.com/lyft/amundsenfrontendlibrary/pull/298)

### Tests

Unit tests for table and column description upsertion have been modified

### Documentation

Markdown description format demo:
https://rexxars.github.io/react-markdown/

### CODE REVIEW

**Comment by feng-tao** [comment]
Seems pretty safe. In the mean time, could you also fix the CLA check? Normally it is due to you didn't configure your github handle(username, email) in your local box.
cc @jinhyukchang

File: metadata_service/api/column.py [hunk 1]
```diff
@@ -21,16 +21,18 @@ def __init__(self) -> None:
...
+            column_name: str) -> Iterable[Union[dict, tuple, int, None]]:
         """
-        Updates column description
+        Updates column description (passed as a request body)
+        :param table_uri:
+        :param column_name:
+        :return:
         """
         try:
+            description = self.parser.parse_args()['description']
```
Comment (feng-tao) [inline metadata_service/api/column.py]: @verdan , I assume it won't affect the atlas backend?

Comment (verdan) [reply metadata_service/api/column.py]: It shouldn't. go ahead!!

**Comment by feng-tao** [comment]
the backend code looks fine. So once we are agree with the FE part, we are good to go with this pr. And please increase the version as well.

**Comment by Mikhail-Ivanov** [comment]
Please note that I've got rid of reqparse because of this (also faced issues while writing unit tests):
https://stackoverflow.com/questions/19384526/how-to-parse-the-post-argument-to-a-rest-service

File: metadata_service/api/column.py [hunk 2]
```diff
@@ -13,24 +15,22 @@ class ColumnDescriptionAPI(Resource):
...
+            column_name: str) -> Iterable[Union[dict, tuple, int, None]]:
         """
-        Updates column description
+        Updates column description (passed as a request body)
+        :param table_uri:
+        :param column_name:
+        :return:
         """
         try:
+            description = json.loads(request.json).get('description')
```
Comment (ttannis) [inline metadata_service/api/column.py]: What do you all think about having a check for what happens if the description is `None`? Does `self.client.put_column_description` end up handling this, or will the description be deleted?

Comment (Mikhail-Ivanov) [reply metadata_service/api/column.py]: I've checked it locally (via Postman):
1) {"description": ""} - this request payload will truncate description
2) request payload with no "description" tag ends with {"msg": "Encountered exception: None"} and description data stay untouched

Comment (ttannis) [reply metadata_service/api/column.py]: Thanks for confirming the behavior for empty string and `None`. They behave as we expect, so no changes would be needed.

**Pull request merged**